README.md

```diff
@@ -1,2 +1,9 @@
-# review_past_project
-A mock review based on the Microverse 2.0 review standards.
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+
+# Note
+
+This repository was forked in order to complete the [Microverse trials](https://support.microverse.org/en/articles/3047017-what-are-the-microverse-trials). The previous section came with the original repository and was left here for informative purposes.
```

Comment:
This README file it's great, but it's necessary to have more information for non-technical people.
The README haves the old information. 
Would be great to have fully information about your newer project. Like: Title, description, live view link, an image, who created the project and a license.